src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d.MM.yyyy");
```

Comment:
if we use this variable only in this class let's make it private

Reply:
done